Flank/src/main/java/com/walmart/otto/Flank.java

```diff
@@ -233,21 +256,29 @@ private void printShards(Configurator configurator, int numberOfShards) {
 
   private List<String> getTestCaseNames(OptionSet options) {
     System.setOut(getEmptyStream());
-    List<String> filteredTests = new ArrayList<>();
+    List<TestMethod> testMethods = DexParser.findTestMethods((String) options.valueOf("t"));
+    System.setOut(originalStream);
```

Comment:
what's this for?

Reply:
I suppose we want to discard whatever DexParser prints to stdout. That was already in place, I just moved the lines.

Reply:
oh, I didn't notice it was there before. sorry.